web/package.json

```diff
@@ -4,38 +4,38 @@
   "private": true,
   "dependencies": {
     "@equinor/eds-core-react": "^0.28.0",
-    "@equinor/eds-icons": "^0.17.0",
+    "@equinor/eds-icons": "^0.18.0",
     "@equinor/eds-tokens": "^0.9.0",
-    "axios": "^1.2.2",
+    "axios": "^1.3.1",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "react-is": "^18.2.0",
-    "react-oauth2-code-pkce": "^1.10.0",
+    "react-oauth2-code-pkce": "^1.10.1",
     "react-scripts": "5.0.1",
     "styled-components": "^5.3.6"
   },
   "devDependencies": {
     "@testing-library/jest-dom": "^5.16.5",
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^14.4.3",
-    "@types/jest": "^29.2.5",
+    "@types/jest": "^29.4.0",
     "@types/node": "^18.11.18",
-    "@types/react": "^18.0.26",
+    "@types/react": "^18.0.27",
     "@types/react-dom": "^18.0.10",
     "@types/react-router": "^5.1.20",
     "@types/react-router-dom": "^5.3.3",
     "@types/styled-components": "^5.1.26",
-    "@typescript-eslint/eslint-plugin": "^5.47.1",
-    "@typescript-eslint/parser": "^5.47.1",
-    "eslint": "^8.31.0",
-    "eslint-config-prettier": "^8.5.0",
+    "@typescript-eslint/eslint-plugin": "^5.50.0",
+    "@typescript-eslint/parser": "^5.50.0",
+    "eslint": "^8.33.0",
+    "eslint-config-prettier": "^8.6.0",
     "eslint-config-react-app": "^7.0.1",
-    "eslint-plugin-jsx-a11y": "^6.6.1",
+    "eslint-plugin-jsx-a11y": "^6.7.1",
     "eslint-plugin-prettier": "^4.2.1",
-    "eslint-plugin-react": "^7.31.11",
+    "eslint-plugin-react": "^7.32.2",
     "eslint-plugin-react-hooks": "^4.6.0",
-    "prettier": "2.8.1",
-    "typescript": "^4.9.4"
+    "prettier": "2.8.3",
+    "typescript": "^4.9.5"
```

Comment:
Did you have any issues with this one? If I use yarn upgrade-interactive to update this I often get problems with typescript not finding my other packages, ref https://github.com/equinor/mercury/pull/315

Reply:
No issues. "yarn up" did nothing :man_shrugging: